app/presenters/detailed_guide_presenter.rb

```diff
@@ -43,8 +43,35 @@ def related_mainstream
     links("related_mainstream")
   end
 
+  def applies_to
+    return nil if !national_applicability
+    all_nations = national_applicability.values
+    applicable_nations = all_nations.select { |n| n["applicable"] }
+    inapplicable_nations = all_nations - applicable_nations
+
+    applies_to = applicable_nations.map { |n| n["label"] }.to_sentence
+
+    if inapplicable_nations.any?
+      nations_with_alt_urls = inapplicable_nations.select { |n| n["alternative_url"] && !n["alternative_url"].empty? }
+      if nations_with_alt_urls.any?
+        additional_guidance_links = nations_with_alt_urls
+          .map { |n| "<a rel=\"external\" href=\"#{n['alternative_url']}\">#{n['label']}</a>" }
```

Comment:
You could probably use the Rails link_to helper to build this link, that might be cleaner.
See: https://github.com/alphagov/government-frontend/blob/master/app/presenters/linkable.rb#L21-L23